packages/contents/src/drivefs.ts

```diff
@@ -552,6 +552,7 @@ export abstract class ContentsAPI {
 export class ServiceWorkerContentsAPI extends ContentsAPI {
   constructor(
     baseUrl: string,
+    tabId: string,
```

Comment:
Since adding this new argument as the second argument in the list, maybe we should take the opportunity to make `ServiceWorkerContentsAPI` and `ContentsAPI` take an object as `options`, so it's easier to maintain and potentially extend later.

Reply:
Done!